include/pttstruct.h

```diff
@@ -232,6 +232,9 @@ typedef struct boardheader_t { /* 256 bytes */
 #define BRD_ALIGNEDCMT		0x04000000	/* ������� */
 #define BRD_NOSELFDELPOST       0x08000000      /* ���i�ۧR */
 #define BRD_BM_MASK_CONTENT	0x10000000	/* ���\�O�D�R���S�w��r */
+#ifdef POST_COMMENT_WITH_TAIWAN_PHONE_ONLY
+#define BRD_TAIWAN_PHONE_ONLY  0x20000000
```

Comment:
`BRD_NEED_SMS_VERIFICATION`

Also, flags should not be guarded with `#ifdef`s. If the bit is taken, it is taken. This avoids surprises when migrating between different versions and/or build flags.

Reply:
Ok, the `#ifdef` is for previous requirement. so I will remove `POST_COMMENT_WITH_TAIWAN_PHONE_ONLY ` flag